manifests/cli.pp

```diff
@@ -23,11 +23,7 @@
   }
 
   $jar = "${jenkins::libdir}/jenkins-cli.jar"
-  $extract_jar = "jar -xf ${jenkins::libdir}/jenkins.war WEB-INF/lib/"
-  $move_jar = "mv WEB-INF/lib/cli-*.jar ${jar}"
-  $remove_dir = 'rm -rf WEB-INF'
-  $cli_tries = $jenkins::cli_tries
-  $cli_try_sleep = $jenkins::cli_try_sleep
+  $extract_jar = "/bin/wget -O ${jar} http://127.0.0.1:8080/jnlpJars/jenkins-cli.jar"
```

Comment:
The host part of the URL should be `localhost`, not `127.0.0.1`, so that we are not dependent on legacy IPv4 being available.